src/components/application_manager/src/mobile_message_handler.cc

```diff
@@ -104,7 +104,7 @@ application_manager::Message* MobileMessageHandler::HandleIncomingMessageProtoco
   }
   LOG4CXX_DEBUG(logger_, "Incoming RPC_INFO: " <<
                          (out_message->connection_key() >> 16) <<", "<<
-                         GetMessageTypes[out_message->type()] <<", "<<
+                         //GetMessageTypes[out_message->type()] <<", "<<
```

Comment:
Usually commented code is not allowed. Later nobody will know is it ok to uncomment it. Is it ok to have it here?